### PR TITLE
CORDA-3069: Allow SLF4J to initialise within the sandbox.

### PR DESCRIPTION
**Pull request by chrisr3**
- Provide the sandbox example with a NOP implementation of SLF4J's `StaticLoggerBinder` class.
- Allow `SandboxClassLoadingException` instances to escape the sandbox unhindered.
- Stub out `System.setSecurityManager()` always to throw `AccessControlException`.

### CODE REVIEW

File: djvm/src/main/kotlin/net/corda/djvm/analysis/AnalysisConfiguration.kt [hunk 1]
```diff
@@ -377,6 +377,15 @@ class AnalysisConfiguration private constructor(
...
             )
+        ).mapByClassName() + listOf(
+            // This method will be deleted.
+            Member(
+                access = ACC_STATIC,
+                className = sandboxed(SecurityManager::class.java),
+                memberName = "<clinit>",
+                descriptor = "()V",
+                genericsDetails = ""
+            )
```
Comment (IgorNitto) [inline djvm/src/main/kotlin/net/corda/djvm/analysis/AnalysisConfiguration.kt]: It would be nice if there was a way to specify the bytecode modifcations that go into the sandbox that is structured more like actual code; a long list of blocks calling a bytecode look-alike DSL becomes very hard to follow very rapidly.

File: djvm-example/settings.gradle [hunk 2]
```diff
@@ -5,3 +5,11 @@ pluginManagement {
 }
 
 rootProject.name = "djvm-example"
+
+if (file('../djvm').isDirectory()) {
```
Comment (IgorNitto) [inline djvm-example/settings.gradle]: This seems unrelated to this change?

File: djvm/src/main/kotlin/net/corda/djvm/analysis/AnalysisConfiguration.kt (hunk 1 above)
Comment (chrisr3) [reply djvm/src/main/kotlin/net/corda/djvm/analysis/AnalysisConfiguration.kt]: Yes, this feature is growing beyond it's original intention.

File: djvm-example/settings.gradle (hunk 2 above)
Comment (chrisr3) [reply djvm-example/settings.gradle]: Umm, well, possibly :wink:. Technically yes, but this and SLF4J initialisation are fixing the common _underlying_ issue that Joel's example functions aren't working correctly in the independent sub-project.

Comment (chrisr3) [reply djvm-example/settings.gradle]: To be fair, the changes to `archiveBaseName` in the `build.gradle` file are related to this Gradle Composite Build change too... :speak_no_evil:
